change/@ni-nimble-tokens-9e9f7223-3c56-4a3e-a547-a00214a35d34.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Remove icon-check and rename icon-check-large to icon-check",
+  "packageName": "@ni/nimble-tokens",
+  "email": "20542556+mollykreis@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

Guess this should be dependentChangeType major since it'll result in a breaking component removal in all frameworks

```suggestion
  "dependentChangeType": "major"
```